modules/bound-sa-tokens-configuring.adoc

```diff
@@ -43,10 +43,10 @@ spec:
 <1> This value should be a URL from which the recipient of a bound token can source the public keys necessary to verify the signature of the token. The default is [x-]`https://kubernetes.default.svc`.
 
 .. Save the file to apply the changes.
++
+// TODO: See if it's okay to bump this up to level 2 numbering instead of under the optional section:
```

Comment:
TODO?

Reply:
This was just a question to myself, on whether it's okay to always have the user wait for the Kube API server pods to restart, even if they don't want to force the holders to request new bound tokens. The bulleted list was just getting indented too far. As long as you're okay with how these steps look, then we're good.